deploy/chart/templates/0000_50_olm_02-services.yaml

```diff
@@ -0,0 +1,39 @@
+{{ if .Values.monitoring.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: olm-operator-metrics
+  namespace: {{ .Values.namespace }}
+  annotations:
+    service.alpha.openshift.io/serving-cert-secret-name: olm-operator-serving-cert
+  labels:
+    app: olm-operator
+spec:
+  type: ClusterIP
+  ports:
+  - name: https-metrics
+    port: 8081
```

Comment:
I need to double-check whether we expose a way to configure the metrics port (e.g. through flags) for the catalog and olm operator binaries. Do we want to update the monitoring helm chart value properties and add a section for configuring this port that defaults to 8081?

Reply:
It looks like it's hardcoded in the olm operator main package. I think exposing a configuration knob makes sense upstream, but it's out of scope for this PR anyways.